.github/workflows/testing.yml

```diff
@@ -26,4 +26,4 @@ jobs:
         - uses: actions/checkout@v2
         - uses: ./
         - name: Check that result file is generated
-          run: ls -l out/analyzer-result.yml
+          run: tree -L 2 .tortellini
```

Comment:
I want to use ls so it fails when the file is not there

Reply:
I'll make it part of #111

Reply:
https://github.com/tortellini-tools/action/pull/111/commits/f2e313c55b5c14b43d58a87c02e00e0a53abf3a0